.gitmodules

```diff
@@ -1,16 +1,16 @@
 [submodule "external-adapters"]
 	path = external-adapters
-	url = https://github.com/hackbg/external-adapters-js
-	branch = feature/terra-ea-refactor
+	url = https://github.com/smartcontractkit/external-adapters-js
+	branch = feature/terra-adapter
```

Comment:
think this one is without locking, I've encountered `sequence mismatch` errors while running the demo. Is that intended?

Reply:
Good finding @denismaxim0v. Added the mutex to this branch.